pyth_observer/check/publisher.py

```diff
@@ -225,11 +237,26 @@ def __init__(self, state: PublisherState, config: PublisherCheckConfig):
         self.__stall_time_limit: int = int(
             config["stall_time_limit"]
         )  # Time in seconds
+        self.__max_slot_distance: int = int(config["max_slot_distance"])
```

Comment:
in this case, max_slot_distance should be added to the config of this check too. Can you add it to the sample?